api-reference/_hidden/dxGanttContextMenuItem/name.md

```diff
@@ -8,3 +8,172 @@ type: String
 Specifies the context menu item name.
 
 ---
+
+---
+
+##### jQuery
+
+    <!--JavaScript-->
+    $(function () {
+        $("#gantt").dxGantt({
+            contextMenu: {
+                items: [
+                    {
+                        text: "Category",
+                        items:[
+                            {
+                                text: "Item 1",
+                                name: "item1"
+                            },
+                            {
+                                text: "Item 2",
+                                name: "item2"
+                            },
+                            {
+                                text: "Item 3",
+                                name: "item3"
+                            }                        
+                        ]                        
+                    }
+                    // ...
+                ]
+            }
+            onCustomCommand: onCustomCommandClick
+            // ...
+        });
+    });
+    function onCustomCommandClick(args) {
```

Comment:
you have _args_ in function parameters, but _e.name_ is used

Reply:
fixed